src/services/Serialization.jl

```diff
@@ -207,3 +211,27 @@ function getTypeFromSerializationModule(dfg::G, moduleType::Symbol) where G <: A
     end
     return st
 end
+
+"""
+$(SIGNATURES)
+Pack a PPE into a Dict{String, Any}.
+"""
+function packPPE(dfg::G, ppe::P)::Dict{String, Any} where {G <: AbstractDFG, P <: AbstractPointParametricEst}
```

Comment:
I hope you like all the packing functions in one place

Reply:
Jislaaik, much better!